### PR TITLE
[UT] add skiplist in pvc w/ lts for skipped cases

### PR DESCRIPTION
**Pull request by AshburnLee**
## Functionality
This PR add all ~~SKIPPED~~ FAILED cases to a skiplist in `scripts/skiplist/pvc_lts/`
language 112 failed cases
operators 17 failed cases

Solves issue https://github.com/intel/intel-xpu-backend-for-triton/issues/986

## Result
~~Now there are no SKIPPED cases on PVC w/ LTS driver~~

## Notes
~~1. This PR only skips SKIPPED cases, no FAILED cases~~
~~2. Agama version is captured form `intel-level-zero-gpu`, because I found the original way won't work on my local PVC w/ LTS, the change in my PR works.~~

### CODE REVIEW

**Comment by AshburnLee** [comment]
About the convert function from agama to driver type, I can not think of another way but hard code the agama version in the function. @vlad-penkin could you give some tips if this is not a good practice?

File: scripts/skiplist/pvc_lts/language.txt [hunk 1]
```diff
@@ -0,0 +1,169 @@
+# https://github.com/intel/intel-xpu-backend-for-triton/issues/986
```
Comment (pbchekin) [inline scripts/skiplist/pvc_lts/language.txt:1]: Why this skip list contains more test cases that the existing default skip list? The run (https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/9214379486) shows that we do not need to skip more tests than the current skip list has.

File: scripts/capture-hw-details.sh [hunk 2]
```diff
@@ -50,3 +46,50 @@ if [ "$QUIET" = false ]; then
     echo "AGAMA_VERSION=$AGAMA_VERSION"
     echo "GPU_DEVICE=$GPU_DEVICE"
 fi
+
+agama_to_driver_type() {
```
Comment (pbchekin) [inline scripts/capture-hw-details.sh:50]: I think we do not need to detect what driver (rolling or lts) is currently installed. Instead, a specific runner should set an environment variable either to select a skip list or to select a release stream (if you still need it for some reason).

Comment (vlad-penkin) [reply scripts/capture-hw-details.sh:50]: @AshburnLee could you please address @pbchekin comments?

**Review by vlad-penkin — CHANGES_REQUESTED** [review]

File: scripts/capture-hw-details.sh (hunk 2 above)
Comment (AshburnLee) [reply scripts/capture-hw-details.sh:50]: Thanks~ I'm working on other PR's comments.

Comment (AshburnLee) [reply scripts/capture-hw-details.sh:50]: So from what I understand, in this PR, I just need to: 
1. remove `agama_to_driver_type()` and `xpu_is_pvc()`, 
2. Run `test_triton.sh` and collect `FAILED` cases from the log, and put those `FAILED` in a skiplist `pvc_lts/....txt`

Please confirm that I am right @pbchekin , thank you.

Comment (AshburnLee) [reply scripts/capture-hw-details.sh:50]: And the automation script https://github.com/intel/intel-xpu-backend-for-triton/pull/1196 is not in this scope

Comment (pbchekin) [reply scripts/capture-hw-details.sh:50]: Yes, you can use test report from https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/9507147050. This is a run where all errors are ignored and full test report is attached as artifact.

```
all: passed: 17460, failed: 129, skipped: 400, xfailed: 1240, total: 19229, fixme: 12 pass rate (w/o xfailed): 97.06%
```

Comment (AshburnLee) [reply scripts/capture-hw-details.sh:50]: working on it.

Comment (AshburnLee) [reply scripts/capture-hw-details.sh]: done~

**Review by pbchekin — APPROVED** [review]

**Comment by pbchekin** [comment]
Implemented in #1456.

**Pull request closed without merging**